Cartfile

```diff
@@ -1,16 +1,16 @@
-github "getsentry/sentry-cocoa"             "4.0.1"
-github "Alamofire/Alamofire"                ~> 4.0
+github "getsentry/sentry-cocoa"             ~> 4.2.1
+github "Alamofire/Alamofire"                ~> 4.8
 github "sleroux/Deferred"                   "Swift3.0"
-github "SnapKit/SnapKit"                    "3.1.2"
-github "rs/SDWebImage"                      "4.1.0"
-github "swisspol/GCDWebServer"              "3.3.2"
-github "kif-framework/KIF"                  "v3.7.3"
+github "SnapKit/SnapKit"                    ~> 4.2.0
```

Comment:
this looks like a major release bump... any breaking changes?

Reply:
Nope, just minor compat changes.